snap/snapcraft.yaml

```diff
@@ -232,8 +232,8 @@ parts:
   runc:
     plugin: make
     source: https://github.com/opencontainers/runc.git
-    # from https://github.com/moby/moby/releases/tag/v24.0.5
-    source-tag: v1.1.12
+    # from https://github.com/moby/moby/releases/tag/v27.0.0-rc.2
```

Comment:
Basing this on the release notes of a RC is questionable. How about linking to https://github.com/moby/moby/blob/v27.2.0/Dockerfile#L290 ?

Reply:
It's not based on an RC, I'm saying that: 

In the RC the version was changed. But the version still the same for the non-rc version. I'm just pointing the place where the version for the Opencontainers get bumped, only for reference. It gets applied for all latter non-rc versions as well. 

The difference is that, once the version is bumped on the e.g `v1.1` it shows up on the release notes, but it didn't show again in the e.g `v1.2`

Reply:
I decision made for an RC may get reverted before the actual release, so this isn't a good reference. 

> The difference is that, once the version is bumped on the e.g v1.1 it shows up on the release notes, but it didn't show again in the e.g v1.2

That's why I don't think it is a good idea to chase release notes while there is exists a canonical reference for all these versions: the Dockerfile.